leaderelection/leader_election.go

```diff
@@ -43,13 +43,46 @@ const (
 
 	DefaultHealthCheckTimeout = 20 * time.Second
 
+	nameLeaseDuration = "LEASEDURATION"
```

Comment:
Maybe change to 
LEADER_ELECTION_LEASE_DURATION
LEADER_ELECTION_RENEW_DEADLINE
LEADER_ELECTION_RETRY_PERIOD

Reply:
name is modeled  from previous const, like "defaultLeaseDuration", would keep it as no change.

Reply:
These variables are setting from OS environment. So we better be explicitly state what they are used for. Otherwise it can be confused for things like "RETRYPERIOD"